lib/lib-auto-generated/src/main/resources/schema/org/sagebionetworks/repo/model/drs/DrsObject.json

```diff
@@ -0,0 +1,64 @@
+{
+	"description": "The object metadata, and a list of access methods that can be used to fetch object bytes.",
+	"properties": {
+		"id": {
+			"type": "string",
+			"description": "An identifier unique to this DrsObject."
+		},
+		"name": {
+			"type": "string",
+			"description": "A string that can be used to name a DrsObject. This string is made up of uppercase and lowercase letters, decimal digits, hypen, period, and underscore [A-Za-z0-9.-_]."
+		},
+		"self_uri": {
+			"type": "string",
+			"description": "A drs URI, as defined in the DRS documentation, that tells clients how to access this object. The intent of this field is to make DRS objects self-contained, and therefore easier for clients to store and pass around."
+		},
+		"size": {
+			"type": "integer",
+			"description": "For blobs, the blob size in bytes. For bundles, the cumulative size, in bytes, of items in the contents field."
+		},
+		"created_time": {
+			"type": "string",
+			"format": "date-time",
+			"description": "Timestamp of content creation in RFC3339."
+		},
+		"updated_time": {
+			"type": "string",
+			"format": "date-time",
+			"description": "Timestamp of content update in RFC3339."
+		},
+		"version": {
+			"type": "integer",
+			"description": "A string representing a version."
+		},
+		"mime_type": {
+			"type": "string",
+			"description": "A string providing the mime-type of the drs object."
+		},
+		"checksums": {
+			"type": "array",
+			"description": "The checksum of the DrsObject. At least one checksum must be provided. For blobs, the checksum is computed over the bytes in the blob. For bundles, the checksum is computed over a sorted concatenation of the checksums of its top-level contained objects.",
+			"items": {
+				"$ref": "org.sagebionetworks.repo.model.drs.Checksum"
+			}
+		},
+		"contents": {
+			"type": "array",
+			"description": "List of object inside bundle.",
+			"items": {
+					"$ref": "org.sagebionetworks.repo.model.drs.Content"
+				}
+		},
+		"access_methods": {
+			"type": "array",
+			"description": "The list of access methods that can be used to fetch the DrsObject. Required for single blobs and bundles have no access method.",
```

Comment:
Descriptions will include text from the design that is specific to the synapse implementation.